tests/test_stdext.ml

```diff
@@ -19,11 +19,11 @@ let find_files () =
     let open Alcotest_ext in
     (check (list path)) name expected actual
   in
-  check ~name:"Path.find_files empty" ~paths:[] ~name_wo_ext:"" ~expected:[];
-  check ~name:"Path.find_files does not contain" ~name_wo_ext:"foo"
+  check ~name:"Path.find_files empty" ~paths:[] ~name_wo_ext:[ "" ] ~expected:[];
```

Comment:
could you also add a test for `~name_wo_ext:[]`

Reply:
Done.